nodebuilder/network/config.go

```diff
@@ -0,0 +1,15 @@
+package network
+
+type Config struct {
+	Network Network
```

Comment:
We already store the network as part of the Store path. What will happen if someone changes the config to mamaki under `celestia-light-arabica`? Don't think we should allow changing the network in the config.

Reply:
The network flags are parsed before creating the store path, see `flags_node.go`. So if they change it in the config, it affects the store path (as long as they don't pass a custom store, ofc)